dbms/src/Common/getNumberOfCPUCores.cpp

```diff
@@ -25,10 +27,21 @@ UInt16 getNumberOfLogicalCPUCores()
     return CPUCores::number_of_logical_cpu_cores;
 }
 
+UInt16 getNumberOfPhysicalCPUCores()
+{
+    return CPUCores::number_of_physical_cpu_cores;
+}
+
 // We should call this function before Context has been created,
 // which will call `getNumberOfLogicalCPUCores`, or we can not
 // set cpu cores any more.
-void setNumberOfLogicalCPUCores(UInt16 max_logical_cpu_cores)
+void setNumberOfLogicalCPUCores(UInt16 number_of_logical_cpu_cores_)
+{
+    CPUCores::number_of_logical_cpu_cores = number_of_logical_cpu_cores_;
+}
+
+void setNumberOfPhysicalCPUCores(UInt16 number_of_logical_cpu_cores_, UInt16 number_of_hardware_physical_cores)
```

Comment:
this function is actually `computeAndSetNumberOfPhysicalCPUCores()` . `setNumberOfPhysicalCPUCores()` is somehow misleading

Reply:
> this function is actually `computeAndSetNumberOfPhysicalCPUCores()` . `setNumberOfPhysicalCPUCores()` is somehow misleading

okk